Civi/Core/Paths.php

```diff
@@ -67,7 +67,7 @@ public function __construct() {
         );
       })
       ->register('wp.backend.base', function () {
-        return array('url' => CIVICRM_UF_BASEURL . 'wp-admin/');
+        return array('url' => CIVICRM_UF_BASEURL . '/wp-admin/');
```

Comment:
@kcristiano  @alifrumin Any chance that changing this is going to have a consequence of making *other* builds (not buildkit) produce urls like `http://example.org//wp-admin`?

If so, might be better to say `rtrim(CIVICRM_UF_BASEURL, '/') . '/wp-admin/'`

Reply:
@kcristiano 3c8872750de4d82d79d1de8f909c40230e1fd564 looks good to me